e2e/components/UnderlineNav.test.ts

```diff
@@ -0,0 +1,159 @@
+import {test, expect} from '@playwright/test'
+import {visit} from '../test-helpers/storybook'
+import {themes} from '../test-helpers/themes'
+import {viewports} from '../test-helpers/viewports'
+
+test.describe('UnderlineNav', () => {
+  Object.keys(viewports).forEach(key => {
+    themes.forEach(theme => {
+      test(`VRT @ ${key} (${viewports[key]}px) on ${theme} theme`, async ({page}) => {
+        await page.setViewportSize({width: viewports[key], height: 768})
+        await visit(page, {
+          id: 'components-underlinenav--internal-responsive-nav',
+          globals: {
+            colorScheme: theme
+          }
+        })
+        expect(await page.screenshot()).toMatchSnapshot()
+      })
+    })
+  })
+})
+
+test.describe('UnderlineNav Interaction', () => {
+  test('Internal Responsive Nav', async ({page}) => {
+    await visit(page, {
+      id: 'components-underlinenav--internal-responsive-nav'
+    })
+
+    // Default state
+    expect(await page.screenshot()).toMatchSnapshot()
+
+    await page.setViewportSize({width: viewports['primer.breakpoint.sm'], height: 768})
+
+    // Resize
+    expect(await page.screenshot()).toMatchSnapshot()
+
+    await page.getByRole('button', {name: 'More'}).click()
+    expect(await page.screenshot()).toMatchSnapshot()
+
+    await page.getByRole('link', {name: 'Settings 10'}).click()
+    expect(await page.screenshot()).toMatchSnapshot()
+  })
+  test('Hide Icons when there is not enough space to display all list items', async ({page}) => {
+    await visit(page, {
+      id: 'components-underlinenav--internal-responsive-nav'
+    })
+
+    // Default State
+    expect(await page.screenshot()).toMatchSnapshot()
+
+    // Resize
+    await page.setViewportSize({width: viewports['primer.breakpoint.md'], height: 768})
```

Comment:
Would it be helpful to run axe at all during things like resize or when the menu is open, or not really?